lib/paypal_express/generate_summary.rb

```diff
@@ -4,22 +4,23 @@
 module FlexCommerce
   module PaypalExpress
     # @class GenerateSummary
-    # 
+    #
     # This class is used while setting up the paypal for FE
     # It deals with line items total, sub total, tax calculations and
     # Also deals with discounted line items and discounts inorder to send to paypal
-    # 
+    #
```

Comment:
This class also does something for gift cards now; it might need to be updated.

Reply:
Thanks. It's simply subtracting gift card amount from total amount but I will update it.